sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -261,6 +255,8 @@ public bool IsRunning
         ///
         private PartitionManager StorageManager { get; }
 
+        private PartitionLoadBalancer LoadBalancer { get; }
```

Comment:
// TODO:  Appease the pedantic bald reviewer with doc comments.   😛

Reply:
:smile: